assets/css/heading.css

```diff
@@ -4,9 +4,12 @@
     color:#adcce9;
 }
 .largest-heading{  
-    font-size:50px;
-    font-weight:400;
+    font-size:60px;
+    /*font-weight:600;*/
     color:#111;
+    font-family: 'Roboto', sans-serif;           
+    src:url('../fonts/Roboto/Roboto-Medium.ttf');
```

Comment:
Po co src? To już ustawiałeś w font face.
Przejrzyj co napisałeś, pogoogluj i spróbuj zrozumieć co i jak